app/Examples/StackPopAlice.hs

```diff
@@ -88,11 +87,11 @@ Alice works on the @pop@ method of a Java class @Stack@.
 example :: (MonadColorPrinter m) => State UUID (Example m SimpleJavaGrammar String)
 example =
     do
-        tree_start <- StackPopAlice.startTree
-        tree_cond <- StackPopAlice.condTree
-        tree_clonedef <- StackPopAlice.cloneDef
-        tree_clonestorage <- StackPopAlice.cloneStorage
-        tree_cloneretstatement <- StackPopAlice.cloneRetStatement
+        tree_start <- startTree
+        tree_cond <- condTree
+        tree_clonedef <- cloneDef
+        tree_clonestorage <- cloneStorage
+        tree_cloneretstatement <- cloneRetStatement
```

Comment:
Note that this is in Alice's example file.
But, I can change this if you like, its just a bit ugly because you cannot import your own module with a shortened name, so it will then read `Examples.StackPopAlice.condTree` whereas in Bob's example it's simply `StackPopAlice.example`.

Reply:
Oops. Thought, it would be Bob's file. 😅 Then leave as is.